omero/users/cli/import.txt

```diff
@@ -140,6 +154,38 @@ Some of the options available to the import command are:
 
         $ bin/omero import broken_image --report --logs --email my.email@domain.com
 
+Advanced Import Commands
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. option:: --java-help
+
+    Display the help for the Java options of the import command
+
+    Java options can be passed after ``--``, e.g.::
+
+       $ bin/omero import image.tif -- --name=test --description=description
+
+    The above command will import the image "image.tif" with the name "test" and with a
+    description: "description". Visit :ref:`container-annotations` to get a basic overview of how 
+    annotations can be created and linked to OMERO objects (an image, in this case).
+
+.. option:: --advanced-help
+
+    Display the advanced help for the import command, e.g.:: 
```

Comment:
These nested literal blocks are what is upsetting the build, you don't need the `::` here because you have normal text below and only want the `$ bin/omero` sections in codeblocks.